<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#353935;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_plus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_minus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#71797E;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_multiple">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#F28C28;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_egal">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QPushButton {&lt;/p&gt;&lt;p&gt;    background-color: #007BFF;&lt;/p&gt;&lt;p&gt;    color: white;&lt;/p&gt;&lt;p&gt;    border: none;&lt;/p&gt;&lt;p&gt;    padding: 10px;&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;p&gt;QPushButton:hover {&lt;/p&gt;&lt;p&gt;    background-color: #0056b3;&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;p&gt;QPushButton:pressed {&lt;/p&gt;&lt;p&gt;    background-color: #004085;&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#F28C28;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_divide">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#F28C28;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>348</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color:#71797E;
	color:white;
	border:0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_clear">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#F28C28;
	padding:5px;
	border:0;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border:0;
	color:white;
	background-color:transparent;
	font: 700 italic 12pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string> History</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="res1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color:#71797E;
	color:white;
	border:0;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="res2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color:#71797E;
	color:white;
	border:0;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="res3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color:#71797E;
	color:white;
	border:0;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="res4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color:#71797E;
	color:white;
	border:0;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
